airflow/models/abstractoperator.py

```diff
@@ -386,7 +411,9 @@ def unmap(self, resolve: None | dict[str, Any] | tuple[Context, Session]) -> Bas
         """
         raise NotImplementedError()
 
+    @final
```

Comment:
One small thing, I guess it is not intended to overwrite this property. So I add type finalize type annotation to this property

Reply:
Agreed.